src/core/CCFormat.mli

```diff
@@ -65,8 +66,8 @@ val seq : ?sep:unit printer -> 'a printer -> 'a sequence printer
 
 val opt : 'a printer -> 'a option printer
 (** [opt pp] prints options as follows:
-    [Some x] will become "some foo" if [pp x ---> "foo"].
-    [None] will become "none". *)
+    - [Some x] will become "some foo" if [pp x ---> "foo"].
+    - [None] will become "none". *)
```

Comment:
Instead of:
```
opt pp prints options as follows: Some x will become "some foo" if pp x ---> "foo". None will become "none".
```
ocamldoc & odoc display:
```
opt pp prints options as follows:

. Some x will become "some foo" if pp x ---> "foo".
. None will become "none".
```
Seems more readable, to me at least !